core/src/main/java/com/scalar/db/common/error/CoreError.java

```diff
@@ -941,6 +941,18 @@ public enum CoreError implements ScalarDbError {
       "Handling the before-preparation snapshot hook failed. Details: %s",
       "",
       ""),
+  DATA_LOADER_ERROR_CRUD_EXCEPTION(
+      Category.INTERNAL_ERROR,
+      "0047",
+      "something went wrong while trying to save the data",
```

Comment:
```suggestion
      "Something went wrong while trying to save the data",
```

Reply:
[This error code document](https://github.com/scalar-labs/docs-scalardb/blob/fdb2fa2969c5c799721eea96f42123aafe44e4d7/docs/scalardb-core-status-codes.mdx#core-20003) is build from these message definitions in `CoreError`. I think `Details: %s` is needed as well as other errors from the perspective of documentation.